src/router/paths.ts

```diff
@@ -26,3 +26,24 @@ export function navigate(path: PathType) {
   }
   currentPathPrivate.set(path);
 }
+
+const messageFormatter = get(format);
+const appName = 'micro:bit machine learning tool';
+export const getTitle = (path: PathType, t: typeof messageFormatter) => {
```

Comment:
Maybe this should just be `: any`. The i18n package doesn't export the MessageFormatter type (I don't think) so this is a workaround for that.

Reply:
Replicated the type we care about -> [f72eb2d](https://github.com/microbit-foundation/ml-trainer/pull/165/commits/f72eb2dad4edd3090078cde6496d68ce8c987bc9).